people/salary.md

```diff
@@ -2,22 +2,24 @@
 
 ## Setting Your Salary
 
-Because we live in different parts of the world, have different lifestyles and different tax laws it's almost impossible for the company to set a fair salary for everyone. Therefore, **we give the biggest weight to your own proposal** for your salary. We have a maximum salary cap which is five times that of the current minimum salary in the company. The founders' salaries are published here as a benchmark.
+Because we live in different parts of the world, have different lifestyles and different tax laws it's almost impossible for the company to set a fair salary for everyone. Therefore, **we give the biggest weight to your own proposal** for your salary. The founders' salaries are published below for orientation.
+
+We have a salary cap which is five times that of the current minimum salary in the company because we hate how CEOs of some corporations [earn hundreds of times more](https://www.cnbc.com/2018/01/22/heres-how-much-ceo-pay-has-increased-compared-to-yours-over-the-years.html) than their minimum wage workers. 
 
 As a company, we made the decision ([IRL#4]) to have an open salary policy in order to promote equality and transparency. There is no overall list for quick comparison instead the salary and bonuses of each Nitean are available in our Scrooge salary app to be viewed on an individual basis.
 
 We welcome questions about salaries, bonuses, and responsibilities to help you decide what would be a fair salary. Everyone is also free to propose that someone should have a higher/lower salary or that we should increase minimum salary (which is the only way to increase the maximum salary).
 
 
-## Current Company Salaries (January 2017)
+## Current Company Salaries (June 2018)
 
 All amounts are gross (total cost to the company). Some people take care of their own taxes and for some the company pays them, however the gross salary is the same.
 
-**Minimum salary:** 650 EUR
+**Minimum salary:** 1300 EUR
 
-**Maximum salary:** 3250 EUR
+**Founders' salary:** 2500 EUR (seniors are paid more)
 
-**Founder's salary:** 2500 EUR
+**Salary cap:** 6500 EUR ([factor of 5](https://github.com/niteoweb/handbook/blob/master/people/salary.md#setting-your-salary))
```

Comment:
Do we have a style guide for Handbook somewhere that I miss? Since when are we requiring all links to be referenced in the footer?

Which in itself is a bad idea if you ask me. It makes it much harder to spot bad/broken/old links because they are detached from their context. However if you keep the full URL where the link is used, you'll see it the next time you edit the paragraph and you're more likely to notice if the link is outdated.

Reply:
The style guide is an unfinished story: #125 I was given editorial responsibility for the handbook and I consider the Wikipedia style guide to be our reference point. I have referenced this before in handbook changes I have made.

It makes it a lot easier to spot a bad link in a nice formatted list than in the middle of a sentence and I highly doubt most links will be spotted unless clicked upon so that's a moot point. Also, we (will) have a sphinx link checker for that responsibility. As I said in the other comment this makes editing, reading and git diffs much easier with the link references in the footer of the page and is already in use in the handbook, especially for long links such as these.

Reply:
1. Both @dmurko and I are against banning Title case: https://github.com/niteoweb/handbook/issues/125#issuecomment-371788105
2. Much more than certain styles I care about consistency. If we ban Title Case from the Handbook we need to ban it from everywhere else too. 
3. The style you linked is exactly that: unfinished. 
4. From the article you linked: `the pros and cons of using sentence case or title case in headings`. In headings. Let me repeat: **in headings**. Are we talking about headings or all text here?

1. Does link checker do actual requests or how does it check for links?

Reply:
Done.